README.md

```diff
@@ -45,4 +48,40 @@ For more information on `deltalayer`, please refer to
 
 #### Under Development: terra-gcs-bq-streaming-functions
 
-`GcsBQ` is our skeleton code for `GCS` to `BQ` Streaming. 
\ No newline at end of file
+`GcsBQ` is our skeleton code for `GCS` to `BQ` Streaming. 
+
+# terra-gcs-bq-streaming-functions
+This library is used for Gcs to BQ Streaming Cloud Functions.
+
+Although there is only one cloud function (GcsBQ.java) at the moment. Both `build.gradle` and `cloudbuild.yaml` can easily be extended to build and package multiple Uber JARs for different cloud functions.
```

Comment:
We also have `ProtoApp` now.  Let's make the doc consistent with wherever we end up in the code.

Reply:
Updated the `README` to reflect current state of development.